pkglistgen/tool.py

```diff
@@ -431,15 +431,17 @@ def create_weakremovers(self, target, target_config, directory, output):
 
                 pool.createwhatprovides()
 
+                accepted_archs = set(self.all_architectures)
+                accepted_archs.add('noarch')
+
                 for s in oldsysrepo.solvables_iter():
-                    if s.arch == 'src':
+                    if not s.arch in accepted_archs:
```

Comment:
`not in`

Reply:
There are 127 of such uses - that's E713 btw

Reply:
Is it at least consistently wrong?